kit/score/score.proto

```diff
@@ -2,27 +2,30 @@ syntax = "proto3";
 package score;
 option go_package = "github.com/quickfeed/quickfeed/kit/score";
 
+import "google/protobuf/timestamp.proto";
 import "patch/go.proto";
 
 // Score give the score for a single test named TestName.
 message Score {
-    uint64 ID = 1;
-    uint64 SubmissionID = 2 [(go.field) = {tags: 'gorm:"foreignKey:ID"'}];
-    string Secret = 3 [(go.field) = {tags: 'gorm:"-"'}]; // the unique identifier for a scoring session
-    string TestName = 4;     // name of the test
-    string TaskName = 5;     // name of task this score belongs to
-    int32 Score = 6;         // the score obtained
-    int32 MaxScore = 7;      // max score possible to get on this specific test
-    int32 Weight = 8;        // the weight of this test; used to compute final grade
-    string TestDetails = 9;  // if populated, the frontend may display additional details (TODO(meling) adapt to output from go test -json)
+    uint64 ID           = 1;
+    uint64 SubmissionID = 2 [(go.field) = { tags: 'gorm:"foreignKey:ID"' }];
+    string Secret       = 3 [(go.field) = { tags: 'gorm:"-"' }];  // the unique identifier for a scoring session
+
+    string TestName    = 4;  // name of the test
+    string TaskName    = 5;  // name of task this score belongs to
+    int32 Score        = 6;  // the score obtained
+    int32 MaxScore     = 7;  // max score possible to get on this specific test
+    int32 Weight       = 8;  // the weight of this test; used to compute final grade
+    string TestDetails = 9;  // if populated, the frontend may display these details
 }
 
 // BuildInfo holds build data for an assignment's test execution.
 message BuildInfo {
-    uint64 ID = 1;
-    uint64 SubmissionID = 2 [(go.field) = {tags: 'gorm:"foreignKey:ID"'}];
-    string BuildDate = 3;
-    string BuildLog = 4;
-    int64 ExecTime = 5;
-    string SubmissionDate = 6;
+    uint64 ID           = 1;
+    uint64 SubmissionID = 2 [(go.field) = { tags: 'gorm:"foreignKey:ID"' }];
+    string BuildLog     = 3;
+    int64 ExecTime      = 4;
+
+    google.protobuf.Timestamp BuildDate      = 5 [(go.field) = { tags: 'gorm:"serializer:timestamp;type:time"' }];
```

Comment:
See also comment in `types.proto`. Could you also add a doc comment at the top of the file that explains the tag (and other tags):
```proto
[(go.field) = { tags: 'gorm:"serializer:timestamp;type:time"' }];
```
Can be copied from the `types.proto` file.

Reply:
Note that we should use the `'` single quote ticks instead of the `"` because then we can use the `"` inside the tag description... I made this change in a commit just now.